src/sentry/grouping/ingest/seer.py

```diff
@@ -190,6 +190,13 @@ def get_seer_similar_issues(
     stacktrace_string = get_stacktrace_string(get_grouping_info_from_variants(variants))
     exception_type = get_path(event.data, "exception", "values", -1, "type")
 
+    if stacktrace_string == "":
```

Comment:
is this the right place to add this check? i just worry that on metrics / reporting, we may end up having this flow incorrectly marked as "called seer".

Reply:
great callout, thank you! you're right, I'll need to determine a better spot for this

Reply:
should probably be in the `should_call_seer` function, or something like that. and if we don't want to duplicate work, we can change the signature of this method to take in the formatted stacktrace.

Reply:
yup, that's exactly what I'm doing - thank you!

Reply:
might also be worth adding a metric to keep track of this case! thank you!